auth/src/test/java/org/aerogear/mobile/auth/credentials/OIDCCredentialsTest.java

```diff
@@ -71,7 +72,13 @@ public void testSerialize() throws JSONException {
         assertEquals(serializedCredential.get("authState"), CREDENTIAL_AUTH_STATE);
     }
 
-    @Test
+    @Test(expected = UnsupportedOperationException.class)
+    public void testRenew() throws AuthenticationException {
```

Comment:
Does it make sense to create a test for a 'not yet implemented' method?

Reply:
IMO this enforces us to write a test, once implemented :-)